### ISSUE TITLE
BUG: show address in correct order for country

### ISSUE THREAD

**Issue by iTitus** [Segelzwerg/Rechnung#125]
Some countries require the address fields to be in a specific order.

(Postcode before or after city, state info after country etc.)

**Comment by Segelzwerg** [comment]
Yes, great. We would need to add method writing the address that knows different formats.
